### ISSUE TITLE
A-frame a-text issue

### ISSUE THREAD

**Issue by saurabhburadkar** [aframevr/aframe#5201]
**Description:**

- A-Frame Version: 1.3.0
- Platform / Device: windows/ chrome browser
- Reproducible Code Snippet or URL: https://whip-torpid-sousaphone.glitch.me/test4.html

Text not rendering correctly in Aframe.
I tried both a-text and a-entity-text

![Screenshot 2023-01-03 at 2 49 36 PM](https://user-images.githubusercontent.com/71985034/210329560-e77b6e20-7d91-44cd-a693-f0138894e5a3.png)

**Comment by dmarcos** [comment]
Your example is using A-Frame 1.2.0. Can you reproduce with 1.4.0 that it's the latest version?

**Comment by saurabhburadkar** [comment]
its A-frame issue, right?
Because I haven't seen any error logged in console.

**Comment by dmarcos** [comment]
FWIW, text is rendering fine for me on MacOS `Version 108.0.5359.124 (Official Build) (arm64)
` 

![text](https://p199.p4.n0.cdn.getcloudapp.com/items/Z4um1R8X/0f7152ed-bd07-45bb-9020-88d576b04ec7.jpg?source=viewer&v=21ee9d8dec88db7f6b2429464f91f97e)

**Comment by saurabhburadkar** [comment]
yes on mac OS its working fine, but on windows chrome its not working.

**Comment by saurabhburadkar** [comment]
I have macOS system too, but for windows testing I used browserstack site and its showing same issue.
Also I tested this on real system too and same issue

**Comment by saurabhburadkar** [comment]
If you need my help to debug this issue, I am always available here to help you

**Comment by dmarcos** [comment]
What about other browsers on Windows besides Chrome?

**Comment by saurabhburadkar** [comment]
edge browser 108 - not working 
firefox - 108 - working 
opera 94 - not working

**Comment by saurabhburadkar** [comment]
![image](https://user-images.githubusercontent.com/71985034/213690777-4b5f6239-c4d9-4507-9c6e-ccd22ac2c179.png)
same issue in 1.0.4 version

**Comment by dmarcos** [comment]
This has to be something specific to Windows OS: drivers, WebGL shaders implementation... Don't have a Windows machine to investigate. PRs always welcome

**Comment by saurabhburadkar** [comment]
![photo_2023-01-24_10-20-39](https://user-images.githubusercontent.com/71985034/214214655-9b1cc44f-f307-4a5c-b414-246532c9dbd5.jpg)
This is what the console logs show. I don't have a Windows system but for testing, I used the BrowserStack site. They are providing virtual machines for live testing.

**Comment by Omegahed** [comment]
I just tried to reproduce the bug using the following link: https://whip-torpid-sousaphone.glitch.me/test4.html -- at the time of testing:

A-Frame Version | Platform | Version | Browser | Version | Result
--- | --- | --- | --- | --- | ---
1.4.0 | Windows 10 Pro | Version 22H2 (19045.2604) | Chrome | 110.0.5484.100 (64-bit) | Passed
1.4.0 | Windows 10 Pro | Version 22H2 (19045.2604) | Edge | 110.0.1587.41 | Passed
1.4.0 | Windows 10 Pro | Version 22H2 (19045.2604) | Chrome | 109.0.1 | Passed
1.4.0 | Windows 10 Pro | Version 22H2 (19045.2604) | Opera | 95.0.4635.37 | Passed
1.4.0 | iOS | 16.2 | Safari | 16 | Passed

I'm unable to reproduce the artifacting / text rendering bug shown.

@saurabhburadkar do you have any other reproduction steps or code that might further demonstrate the bug?

**Comment by dmarcos** [comment]
Probably issue specific to gpu / driver combo.

**Comment by crcdng** [comment]
Actually, I have apparently the same issue on Mac. 

MacBook Pro Intel, MacOS 14.6, Chrome 127.0.6533.100

` <a-text value="Now Make a VR!" color="#000" position="-1.3 1 2" width=8></a-text>`
<img width="831" alt="Screenshot 2024-08-07 at 16 41 45" src="https://github.com/user-attachments/assets/3722d2dd-eebf-4a99-91e6-2d0063f86c7e">
